R/resource.R

```diff
@@ -0,0 +1,160 @@
+##' Specify what resources a task requires to run.
+##'
+##' # Windows cluster
+##'
+##' The windows cluster currently is all 32-core nodes, with RAM
+##' mainly 512Gb, and a few 384Gb.
```

Comment:
You promised to mention available queues here. You might also translate these numbers here into options:

> This means
> 
> * `cores` can take any integer value from 1 to 32 inclusive
> * `memory_per_node` and `memory_per_process` can range up to `512Gb`, but past `384Gb` you may start limiting the nodes you can use